metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -409,6 +424,14 @@ def get_inspectors(self) -> Iterable[Inspector]:
             inspector = inspect(conn)
             yield inspector
 
+    def get_db_name(self, inspector: Inspector) -> str:
+        engine = inspector.engine
+
+        if engine and hasattr(engine, "url") and hasattr(engine.url, "database"):
+            return str(engine.url.database).strip('"').lower()
```

Comment:
What is the risk of lower casing the database? Is it common to do case-insensitive db names in Relational DBs? Maybe add a comment here to note the risk?

Reply:
Image that we lower case, then later decide to _not_ lower case. We may break a bunch of entities that were already created!

Reply:
Currently we lowercase database names (sqlalchemy does this) -> if you check the demo site there everything is lowercased -> https://demo.datahubproject.io/dataset/urn:li:dataset:(urn:li:dataPlatform:snowflake,demo_db.public.%22test-table%22,PROD)/Schema?is_lineage_mode=false
in snowflaek demo_db is DEMO_DB